src/Sulu/Bundle/CategoryBundle/Admin/CategoryAdmin.php

```diff
@@ -28,6 +28,10 @@ public function __construct(SecurityCheckerInterface $securityChecker, $title)
     {
         $this->securityChecker = $securityChecker;
 
+        if (!$this->securityChecker) {
+            return;
```

Comment:
Shouldn't we add `TODO`s here and in the other Admins, so that we know we have to remove that later?

Reply:
Don't think so.. we have to clean up the whole admin before 2.0 release